lib/util/secure_getenv.h

```diff
@@ -0,0 +1,14 @@
+#ifndef HAVE_SECURE_GETENV
```

Comment:
Put this around the function definition only - makes it easy to add more compat stuff later if needed...

Reply:
I wrapped that around the include statement, too. Or is that bad practice?